functions-methods.js

```diff
@@ -8,9 +8,22 @@
 // getEmailDomain("n.eeken@novi-education.nl") geeft novi-education.nl
 // getEmailDomain("t.mellink@novi.nl") geeft novi.nl
 // getEmailDomain("a.wiersma@outlook.com") geeft outlook.com
+function getEmailDomain(arr) {
+    let mailArray = Array.from(arr);
+    let counter = 0;
+    let domain;
+    for (let i = 0; i < mailArray.length; i++) {
+        if (mailArray[i] !== "@") {
+            counter++;
+        } else {
+            domain = arr.slice(counter + 1);
+            break;
+        }
+    }
+    return domain
```

Comment:
Wellicht had je dit iets simpeler aan kunnen pakken met de split-functie
```suggestion
    return arr.split("@")[1];
```